tests/unit/flow/test_flow_visualization.py

```diff
@@ -0,0 +1,20 @@
+import os
+import numpy as np
+
+from PIL import Image
+
+from jina.flow import Flow
+
+
+def test_visualization():
```

Comment:
that's good, now let's try to have some more difficult Flows to test the power it.